### ISSUE TITLE
@zeit/nccを@vercel/nccに切り替える

### ISSUE THREAD

**Issue by mnao305** [mnao305/chrome-extension-upload#8]
https://www.npmjs.com/package/@zeit/ncc
```
@zeit/ncc is no longer maintained. Please use @vercel/ncc instead.
```
だそうです。

**Comment by orestes** [comment]
Should this issue be closed now that #12 was merged?

**Comment by mnao305** [comment]
#12 has not been merged, so this issue has not yet been resolved.
see https://github.com/mnao305/chrome-extension-upload/pull/12#issuecomment-979142669
